aiohttp/helpers.py

```diff
@@ -460,3 +461,46 @@ def requote_uri(uri):
         # there may be unquoted '%'s in the URI. We need to make sure they're
         # properly quoted so they do not cause issues elsewhere.
         return quote(uri, safe=safe_without_percent)
+
+
+class Timeout:
+    """Timeout context manager.
+
+    Useful in cases when you want to apply timeout logic around block
+    of code or in cases when asyncio.wait_for is not suitable.
+
+    :param timeout: time out time in seconds
+    :param raise_error: if set, TimeoutError is raised in case of timeout
+    :param loop: asyncio compatible event loop
+    """
+    def __init__(self, timeout, *, raise_error=False, loop=None):
```

Comment:
`raise_error` should be `True` by default (`asyncio.wait_for()` raises exception on timeout).
After thinking I doubt do we need the parameter at all.